content_sync/pipelines/definitions/concourse/mass-build-sites.yml

```diff
@@ -203,14 +203,16 @@ jobs:
               cd $CURDIR
               echo "STARTING S3 SYNC FOR $NAME" > /dev/tty
               # START OFFLINE-ONLY
-              zip $NAME.zip -r $SHORT_ID/public
+              cd $CURDIR/$SHORT_ID/public
+              zip $NAME.zip -r ./
               PUBLISH_S3_RESULT=$(aws s3((cli-endpoint-url)) sync ./ s3://((ocw-bucket))$PREFIX/$BASE_URL --exclude="*" --include="$NAME.zip" --metadata site-id=$NAME --only-show-errors) || PUBLISH_S3_RESULT=1
               if [[ $PUBLISH_S3_RESULT == 1 ]]
               then
                 echo "SYNCING $NAME.zip to s3://((ocw-bucket))$PREFIX/$SITE_URL failed for $NAME" > /dev/tty
```

Comment:
I'm not seeing this line get triggered, and there are no zip files in my minio bucket for any courses.  Am I missing some configuration/env setting?

```
STARTING PUBLISH OF course1
PULLING IN STATIC RESOURCES FOR course1
RUNNING HUGO BUILD FOR course1
STARTING S3 SYNC FOR course1
PUBLISH OF course1 COMPLETE
STARTING PUBLISH OF new-site-2
PULLING IN STATIC RESOURCES FOR new-site-2
RUNNING HUGO BUILD FOR new-site-2
STARTING S3 SYNC FOR new-site-2
PUBLISH OF new-site-2 COMPLETE
STARTING PUBLISH OF ocw-www
PULLING IN STATIC RESOURCES FOR ocw-www
RUNNING HUGO BUILD FOR ocw-www
Error: Unable to locate config file or config directory. Perhaps you need to create a new site.
       Run `hugo help new` for details.

HUGO BUILD FAILED FOR ocw-www
PUBLISH OF ocw-www FAILED
FAILED BUILDS HAPPENED
```

Reply:
Deleted all pipelines and tried again, seeing the zips now.  One question - would it be better to use the site url as the root instead of short_id, or does it not really matter as long as there is some root?